htmlelements-matchers/src/main/java/ru/yandex/qatools/htmlelements/matchers/decorators/ConditionMatcherDecorator.java

```diff
@@ -40,9 +49,10 @@ protected void describeMismatchSafely(T item, Description mismatchDescription) {
     }
 
     @Factory
-    public static <T, E> Matcher<T> decorateMatcherWithCondition(final Matcher<? super T> matcher,
-                                                                 final E itemToMatchCondition,
-                                                                 final Matcher<? super E> condition) {
+    public static <T, E> Matcher<T> decorateMatcherWithCondition(
```

Comment:
Nothing wrong, of course it's just formatting, however I reformatted it as I felt it was more readable and sustainable. If you have the time, this is a wonderful talk: https://www.youtube.com/watch?v=oyyFKHpzL0Q